db/dcrsqlite/apisource.go

```diff
@@ -1075,6 +1075,9 @@ func (db *wiredDB) GetExplorerTx(txid string) *explorer.TxInfo {
 		})
 	}
 	tx.Vin = inputs
+	if tx.Vin[0].IsCoinBase() {
+		tx.Type = "Coinbase"
+	}
 	if tx.Type == "Vote" || tx.Type == "Ticket" || tx.Vin[0].IsCoinBase() {
```

Comment:
Why does this block evaluate for coinbase?

Coinbase tx types should determine maturity against this parameter:

https://github.com/decred/dcrd/blob/7647b2594de0cd2a43c2d69a53c8365a6284b08f/chaincfg/params.go#L405-L407

Reply:
Oh, i wasn't trying to evaluate for coinbase maturity, Just wanted block reward to have it own type called coinbase instead of being classified as a regular transaction

Reply:
Right, I get that this wasn't your bit, but I noticed that the block below your edit is evaluated for coinbase although that doesn't make sense.  Is it clear what needs to be changed so the rest of the logic in `GetExplorerTx` makes sense?